addons/beehave/plugin.gd

```diff
@@ -7,8 +7,8 @@ var frames: RefCounted
 
 func _init():
 	name = "BeehavePlugin"
-	add_autoload_singleton("BeehaveGlobalMetrics", "./metrics/beehave_global_metrics.gd")
```

Comment:
this does not quite work always, as Godot reads this as `res://addons/beehave/./metrics/beehave_global_metrics.gd` which can confuse the engine.